src/scenes/ProgressReports/EditForm/Steps/ExplanationOfProgress/ExplanationOfProgress.tsx

```diff
@@ -169,7 +169,18 @@ export const ExplanationOfProgress: StepComponent = ({ report }) => {
               <EnumField
                 name="reasons"
                 label="Select a reason"
-                options={optionsByGroup[group]}
+                options={() =>
+                  optionsByGroup[group].map((reason: any) => (
```

Comment:
```suggestion
                  optionsByGroup[group].map((reason) => (
```
shouldn't need the `any` here, nor is it a good type ;)